home/utils.py

```diff
@@ -77,34 +80,109 @@ def clean_row(row):
             "web_body",
             "whatsapp_title",
             "whatsapp_body",
-            "parent",
+            "viber_title",
+            "viber_body" "messenger_title",
+            "messenger_body" "parent",
```

Comment:
I think you're missing some commas on this line and the one above?

Reply:
weird, I deeeefinitely fixed this, I will fix it again quick